pkg/mkimage-aws/Makefile

```diff
@@ -0,0 +1,19 @@
+IMAGE=mkimage
```

Comment:
this should be `mkimage-aws` now

Reply:
actually, this file should be deleted now!

Reply:
That’s what I get for doing PRs late at night! I’ll update shortly.

Reply:
Pushing...